routes/home/components/Home.js

```diff
@@ -1,28 +1,29 @@
 const React = require('react');
 const T = require('prop-types');
 const { default: Styled } = require('styled-components/native');
-const { View, Image, Button } = require('react-native');
-const { Text } = require('components/Type');
+const { Image } = require('react-native');
+const { Button, Text } = require('@ui-kitten/components');
 const Duck = require('../../../assets/duck.jpg');
 
 const Ducky = Styled(Image)`
-    margin: ${({ theme }) => theme.spacing(2)}px auto;
+    margin: ${({ theme }) => theme.spacing(4)}px auto;
 `;
 
 const WelcomeText = Styled(Text)`
+    margin: ${({ theme }) => theme.spacing('auto', 'auto', 1)};
+`;
+
+const CenteredButton = Styled(Button)`
     margin: auto;
 `;
 
 module.exports = function Home({ navigation }) {
 
-    return <View>
+    return <>
```

Comment:
Guessing this won't come up a whole lot, but is it important that these "base view" (not sure on terms...components passed to `Layout` in the routing config) don't use `react-native's `View` as a root component / at all, since they're wrapped in the `KeyboardAvoidingView`?

If so, is this something we need to take care to enforce? Or is `react-native` pretty vocal if you nest `View`s (I'm assuming that's not something you should do?)?

Reply:
`View` is essentially equivalent to `div` and it's chill to go buck wild and use them wherever. I just removed it here because it wasn't really necessary.

Reply:
oh, sweet, good to know